_docs/geyser/extensions.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Geyser Extensions
+---
+
+## Geyser Extensions
+Geyser Extensions are the equivalent of "plugins", but specifically for the Geyser platform. This brings the advantage of them being platform-agnostic, meaning that you won't have to worry about supporting all platforms individually. Additionally, they will be, by design, only applied for Bedrock players joining via Geyser.
+
+### What can Geyser Extensions do?
+Extensions can fully utilize the Geyser API to add additional functionality to Geyser. See [Geyser API docs](/geyser/geyser-api/) for info on what is currently possible. To name a few examples:
+- Register custom items
+- Hide commands from being suggested
+- Change the MOTD
+- Register your own commands
+- Listen to events, e.g. the emote event.
+
+The underlying Geyser API is steadily expanding, creating more and more opportunities.
+
+### Which Geyser Extensions exist?
+At the moment, not a lot. This is due to Extensions being quite new, and that most stuff can already be done with "normal" plugins/mods.
+There is an [official list](https://github.com/GeyserMC/GeyserExtensionList) of available extensions you can check out.
+
+### Installing Extensions
+To install an extension, simply put the extension .jar file into Geyser's 'extensions' folder. Then, restart Geyser (or the server Geyser runs on).
+
+### Creating Geyser Extensions
+The easiest way to create an extension would be using [this official template](https://github.com/GeyserMC/GeyserExampleExtension/). Simply create a new repository from the template, customize the 'extension.yml' & 'settings.gradle' files, and get started making the extension.
```

Comment:
```suggestion
The easiest way to create an extension would be cloning [this official template](https://github.com/GeyserMC/GeyserExampleExtension/). Simply create a new repository from the template, customize the 'extension.yml' & 'settings.gradle' files, and get started making the extension.
```

Reply:
But do you really "clone" a template? Github itself calls it "use this template"